lib/bigcommerce/lightstep/configuration.rb

```diff
@@ -35,6 +35,9 @@ module Configuration
         verbosity: 1,
         http1_error_code: 500,
         http1_error_code_minimum: 500,
+        max_buffered_spans: 1_000,
```

Comment:
What was the size previously? What impact might this change have on services memory usage?

Reply:
@pedelman That is the default size for the lightstep ruby gem.

Reply:
Sounds good, thanks